### DISCUSSION TITLE
Suggestion for a python based pdf-to-markdown

### DISCUSSION THREAD

**Discussion by Guruprasad93** [jzillmann/pdf-to-markdown#57] [Ideas]
Hi! 

I love the current version of pdf to markdown - but I was wondering if there are any plans OR recommendations for a python based pdf-to-markdown as that would really help when we deploy on the backend. 

Any pointers would be highly appreciated!

**Comment by ixiion001** [comment]
Hello,

i really like PDF to Markdown and it's capabilities. And i would like to 2nd the request from the initial poster, would be great to have a Python-based method for PDF to Markdown conversion, particularly useful for batch conversion.

I would appreciate any insights or recommendations on Python tools or libraries that are effective in PDF to Markdown conversion, preserving the original layout and formatting.

Thanks!

**Comment by jzillmann** [comment]
There might be a `js` based tool one day. python no. That would be a complete rewrite.
